src/docfx/lib/LocalizationUtility.cs

```diff
@@ -43,19 +43,24 @@ public static string AddLeftToRightMarker(CultureInfo culture, string text)
             return s_lrmAdjustment.Replace(text, me => $"{me.Groups[1]}{me.Groups[2]}&lrm;{me.Groups[3]}{me.Groups[4]}");
         }
 
-        public static string? GetFallbackDocsetPath(string docsetPath, Repository? repository, PackageResolver packageResolver)
+        public static string? GetFallbackDocsetPath(string docsetPath, Repository? repository, PackagePath? fallbackRepository, PackageResolver packageResolver)
         {
-            if (repository != null)
+            var docsetSourceFolder = repository == null
+                ? docsetPath.Split('/').Last()
```

Comment:
We currently don't support any fallback if loc content does not belong to a repository. So keep the `if (repository is null) return null` behavior.